django/test/runner.py

```diff
@@ -405,15 +415,20 @@ def _init_worker(counter):
         counter.value += 1
         _worker_id = counter.value
 
+    if multiprocessing.get_start_method() == "spawn":
+        process_setup(*process_setup_args)
+        setup_test_environment()
+
     for alias in connections:
         connection = connections[alias]
-        settings_dict = connection.creation.get_test_db_clone_settings(str(_worker_id))
-        # connection.settings_dict must be updated in place for changes to be
-        # reflected in django.db.connections. If the following line assigned
-        # connection.settings_dict = settings_dict, new threads would connect
-        # to the default database instead of the appropriate clone.
-        connection.settings_dict.update(settings_dict)
-        connection.close()
+        if multiprocessing.get_start_method() == "spawn":
+            # Restore initial settings in spawned processes
+            connection.settings_dict.update(initial_settings[str(alias)])
+            if serialized_contents and alias in serialized_contents.keys():
+                connection._test_serialized_contents = serialized_contents[str(alias)]
+        connection.creation.setup_worker_connection(_worker_id)
+        with captured_stdout():
+            call_command("check", verbosity=-1, databases=connections)
```

Comment:
Is `verbosity=-1` valid? I thought it was one of `{0, 1, 2, 3}`?

Reply:
You are right. 

Looking at this I think we may as well remove the `verbosity` argument. (I'm not sure it applies to "check"?)